{
  "tweet": {
    "edit_info": {
      "initial": {
        "editTweetIds": [
          "1534235031840989184"
        ],
        "editableUntil": "2022-06-07T18:35:10.000Z",
        "editsRemaining": "5",
        "isEditEligible": true
      }
    },
    "retweeted": false,
    "source": "<a href=\"https://hypefury.com\" rel=\"nofollow\">Hypefury</a>",
    "entities": {
      "hashtags": [],
      "symbols": [],
      "user_mentions": [],
      "urls": []
    },
    "display_text_range": [
      "0",
      "81"
    ],
    "favorite_count": "3",
    "id_str": "1534235031840989184",
    "truncated": false,
    "retweet_count": "0",
    "id": "1534235031840989184",
    "created_at": "Tue Jun 07 18:05:10 +0000 2022",
    "favorited": false,
    "full_text": "Fun day of exception handling with Axios and Typescript!\n\nIt's quite the journey!",
    "lang": "en"
  }
}